CI: Require at least one code block separator for example files in the Style Checks workflow (#2810)
GenericMappingTools/pygmt: doc/contributing.md

convention is not applied by the code checking tools, but the PyGMT maintainers
will comment on any pull requests as needed.

> When working on a tutorial or a gallery plot, it's a good practice to use code

Reviewer: ```suggestion
When working on a tutorial or a gallery plot, it is good practice to use code
```

block separators to split a long script into multiple blocks. The separators also

> make it possible to run the script like a Jupyter Notebook in some modern text

Reviewer 2: ```suggestion
make it possible to run the script like a Jupyter notebook in some modern text
```

> editors or IDEs. We consistenly use `# %%` as our code block separators (please
> refer to [issue #2660](https://github.com/GenericMappingTools/pygmt/issues/2660)
> for the discussions). We also require at least one separator in all example files.

Reviewer 2: ```suggestion
editors or IDEs. We consistenly use `# %%` as our code block separators (please
refer to [issue #2660](https://github.com/GenericMappingTools/pygmt/issues/2660)
for the discussions) and require at least one separator in all example files.
```
